modules/appeals_api/spec/fixtures/notice_of_disagreements/v0/valid_10182.json

```diff
@@ -14,11 +19,11 @@
           "areaCode": "555",
           "phoneNumber": "8001111"
         },
-        "email": "clause@north.com",
+        "email": "jdoe@example.com",
```

Comment:
In the swagger specs, I noticed a few places where the test user data was manually edited to remove some "silly" values (like Betty Boop etc.). I figured that we could just make those same changes in the fixtures instead (notice I did leave in the utf8 chars in the veteran's name, though, since we do use those to confirm utf8 is working right in PDF generator tests)